File: src/NuclearCell.cpp
<<<<<<< HEAD:src/NuclearCell.cpp
#include "NuclearCell.h"
//Get the row number of the cell
int NuclearCell::getRowIndex(){
    return row;
}

//Get the column number of the cell
int NuclearCell::getColumnIndex(){
    return column;
}

void NuclearCell::setColumnIndex(int column){
	this->column = column;
}

void NuclearCell::setRowIndex(int row){
	this->row = row;
}

//Get the Color of the cell

Constants::NuclearCellStatus NuclearCell::getStatus(){
    return status;
}

void NuclearCell::setStatus(Constants::NuclearCellStatus status){
	this->status = status;
}
=======
#include "NuclearCell.h"
//Get the row number of the cell
int NuclearCell::getRowIndex(){
    return row;
}

//Get the column number of the cell
int NuclearCell::getColumnIndex(){
    return column;
}

void NuclearCell::setColumnIndex(int column){
	this->column = column;
}

void NuclearCell::setRowIndex(int row){
	this->row = row;
}

//Get the Color of the cell

Constants::NuclearCellStatus NuclearCell::getStatus(){
    return status;
}

void NuclearCell::setStatus(Constants::NuclearCellStatus status){
	this->status = status;
}

NuclearCell::NuclearCell(){
}
>>>>>>> 3a79f7dc034bc0d97e0c2265324f9cbf8b747f78:src/NuclearCell.cpp

File: Python/NuclearCell.py
from Constants import Constants

class NuclearCell:
    def __init__(self):
        self.__row = -1
        self.__column = -1
        self.__status = Constants.NuclearCellStatus.EMPTY

    def getRowIndex(self):
        return self.__row
    
    def getColumnIndex(self):
        return self.__column
    
    def getStatus(self):
        return self.__status

    def setRowIndex(self,row):
        self.__row = row

    def setColumnIndex(self,column):
        self.__column = column

    def setStatus(self,status):
        self.__status = status

    
